holoviews/element/path.py

```diff
@@ -162,26 +167,64 @@ class Box(BaseShape):
 
     y = param.Number(default=0, doc="The y-position of the box center.")
 
+    width = param.Number(default=1, doc="The width of the box.")
+
     height = param.Number(default=1, doc="The height of the box.")
 
-    aspect= param.Number(default=1, doc=""""
-        The aspect ratio of the box if supplied, otherwise an aspect
-        of 1.0 is used.""")
+    orientation = param.Number(default=0, doc="""
+       Orientation in the Cartesian coordinate system, the
+       counterclockwise angle in radians between the first axis and the
+       horizontal.""")
+
+    aspect= param.Number(default=1.0, doc="""
+       Optional multiplier applied to the box length to compute the
+       width in cases where only the length value is set.""")
```

Comment:
The first argument corresponds to the width - the problem is that saying you compute the width from the width and aspect is confusing. I wanted the equivalent of 'diameter' for squashing a circle with the aspect. Box 'size' maybe?